gateway/src/cluster/impl.rs

```diff
@@ -210,14 +210,14 @@ impl Cluster {
         .await;
     }
 
-    /// Brings down the cluster, stopping all of the shards that it's managing.
+    /// Bring down the cluster, stopping all of the shards that it's managing.
     pub fn down(&self) {
         for shard in self.0.shards.lock().expect("shards poisoned").values() {
             shard.shutdown();
         }
     }
 
-    /// Brings down the cluster in a resumable way and returns all info needed
+    /// Bring down the cluster in a resumable way and returns all info needed
```

Comment:
missing a period

Reply:
The sentence moves onto the next line, so this is actually fine